spec/features/user_login_signup_with_social_network_spec.rb

```diff
@@ -1,242 +0,0 @@
-require 'rails_helper'
-
-feature 'User logs in or signs up with a social network', js: true do
```

Comment:
Is it expected that this whole file is gone? I feel like it could be made to work?

Reply:
readded and passing 🎉